### DISCUSSION TITLE
SAFE methode RR_T

### DISCUSSION THREAD

**Discussion by metliv** [asreview/asreview#1582] [Q&A]
Hi,
we want to use the SAFE method (Botje & van de Schoot, 2023) for our screening process. I have a question regarding the phase 2 rule number 2 "At least twice the RR_T records have been screened". Does this mean that at least times two the number of RR_T records have to been screened in the total dataset? So if the calculated RR_T = 500, we need to screen at least 1000 records?

**Comment by j0sien** [comment]
Yes that is correct @metliv 
This is to be SAFE to not exclude any relevant records.
However, in case your review is not systematic or in case you have reasons to believe it is not so bad if you miss one or several relevant records, you could of course take a different ratio.
